src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return 40 * days - 50;
```

Comment:
create some variable and use instead of number '40'

Reply:
else you can delete one of your conditions and leave only `return....`. Nothing change